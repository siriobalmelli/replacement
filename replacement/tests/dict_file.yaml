---
# dictionaries from files, nested (worst corner case)
# tests/dict_file.yaml
replacement:
  - text: dict
    input:
      - dict: file
        input: |
          dict.yaml
...
