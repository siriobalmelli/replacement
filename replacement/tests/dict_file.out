---
replacement:
  - meta: dict
    input:
      num: 1.0
      str: hi
  - text: dict
    input:
      # this list will be recursed (a list is not a dictionary)
      - dict: dict
        prep: substitute
        input:
          version: $num $str
      - dict: text
        prep: substitute
        input: |
          ---
          num: $num
          ...
...
